src/scripts/events/master.js

```diff
@@ -37,7 +36,7 @@ function hasEventEmitter(obj) {
  */
 function detachEvents(target) {
 
-    let hasEvents = target && hasEventEmitter(target) && size(target._events);
+    const hasEvents = target && hasEventEmitter(target) && size(target._events);
```

Comment:
We need to check when `_eventsCount` was added because I don't recall it existing when I wrote this originally.